device-types/Cisco/WS-C2960XR-48PS-I.yaml

```diff
@@ -0,0 +1,76 @@
+---
+manufacturer: Cisco
+model: Catalyst 2960XR-24PS-I
+slug: ws-c2960xr-24ps-i
+part_number: WS-C2960XR-24PS-I
+is_full_depth: false
+u_height: 1
+comments: IP Lite feature set
+interfaces:
+  - name: FastEthernet0
+    type: 100base-tx
+    mgmt_only: true
+  - name: GigabitEthernet1/0/1
+    type: 1000base-t
+  - name: GigabitEthernet1/0/2
+    type: 1000base-t
+  - name: GigabitEthernet1/0/3
+    type: 1000base-t
+  - name: GigabitEthernet1/0/4
+    type: 1000base-t
+  - name: GigabitEthernet1/0/5
+    type: 1000base-t
+  - name: GigabitEthernet1/0/6
+    type: 1000base-t
+  - name: GigabitEthernet1/0/7
+    type: 1000base-t
+  - name: GigabitEthernet1/0/8
+    type: 1000base-t
+  - name: GigabitEthernet1/0/9
+    type: 1000base-t
+  - name: GigabitEthernet1/0/10
+    type: 1000base-t
+  - name: GigabitEthernet1/0/11
+    type: 1000base-t
+  - name: GigabitEthernet1/0/12
+    type: 1000base-t
+  - name: GigabitEthernet1/0/13
+    type: 1000base-t
+  - name: GigabitEthernet1/0/14
+    type: 1000base-t
+  - name: GigabitEthernet1/0/15
+    type: 1000base-t
+  - name: GigabitEthernet1/0/16
+    type: 1000base-t
+  - name: GigabitEthernet1/0/17
+    type: 1000base-t
+  - name: GigabitEthernet1/0/18
+    type: 1000base-t
+  - name: GigabitEthernet1/0/19
+    type: 1000base-t
+  - name: GigabitEthernet1/0/20
+    type: 1000base-t
+  - name: GigabitEthernet1/0/21
+    type: 1000base-t
+  - name: GigabitEthernet1/0/22
+    type: 1000base-t
+  - name: GigabitEthernet1/0/23
+    type: 1000base-t
+  - name: GigabitEthernet1/0/24
+    type: 1000base-t
+  - name: GigabitEthernet1/0/25
+    type: 1000base-x-sfp
+  - name: GigabitEthernet1/0/26
+    type: 1000base-x-sfp
+  - name: GigabitEthernet1/0/27
+    type: 1000base-x-sfp
+  - name: GigabitEthernet1/0/28
+    type: 1000base-x-sfp
+console-ports:
+  - name: con 0
+    type: rj-45
+power-ports:
+  - name: PSU0
+    type: iec-60320-c14
+  - name: PSU1
+    type: iec-60320-c14
```

Comment:
PSU's should be modelled as module bays

Reply:
Hey Dan, I just followed how the existing 2960XRs are setup. Do you want me to create the PSU modules as well? Should i update the other XRs to be the same?

Reply:
Hi - would it be worth updating the contribution guide to make sure this is a clearly communicated standard? I don't disagree with the thinking in principle but there are many PR, including recent ones (eg #846 ) where the PSU have not been done as bays.

We just need to make sure its consistent & squares with the current note in the contribution guide:

_"Include only components which are fixed to the chassis. Optional modular components should be omitted from the definition. (Note that this does not exclude field-replaceable hardware that is expected to always be present, such as power supplies.)"_

Reply:
> Hey Dan, I just followed how the existing 2960XRs are setup. Do you want me to create the PSU modules as well? Should i update the other XRs to be the same?

If you want to, yes, that would be ideal

Reply:
> Hi - would it be worth updating the contribution guide to make sure this is a clearly communicated standard? I don't disagree with the thinking in principle but there are many PR, including recent ones (eg #846 ) where the PSU have not been done as bays.

I agree however sometimes the PSUs are fixed, sometimes they are modules.

I will see what I can do with the contrib guide tonight.

> We just need to make sure its consistent & squares with the current note in the contribution guide:
> 
> _"Include only components which are fixed to the chassis. Optional modular components should be omitted from the definition. (Note that this does not exclude field-replaceable hardware that is expected to always be present, such as power supplies.)"_

Reply:
I have updated that device, and was half way through adding some more. If youre happy with this ill update the other XR switches, make the power supply modules and then got some more devices to add.